### ISSUE TITLE
English documentation is missing

### ISSUE THREAD

**Issue by CyrilleB79** [marlon-sousa/EnhancedFindDialog#21]
Hello

When running NVDA in English, in the add-on manager, one cannot access the add-on's documentation.
On the contrary, the add-on's translated documentation is available through add-on manager (e.g. Franch in my case).

The English documentation exists in any case since it is on the GitHub page.

Could you please add it as it is done automatically for the other add-ons based on the add-on template?
Thanks.

**Comment by thgcode** [comment]
I think it might be a bug on the pipeline, since it generated the contributing.html file correctly. I'll take a look tonight on the pipeline and on the addon template to fix it.

**Comment by thgcode** [comment]
Hello!

It seems the bug was on the pipeline. The addon was compiled by Github actions on a Linux system, and since the case of the file on disk and the code were different the pipeline couldn't find the file. I fixed the pipeline and reuploaded the addon to the release manually this time to fix it.

Thanks for the report!

**Comment by CyrilleB79** [comment]
The issue is fixed in version 1.4.1. Closing.
Thanks.
